gobdd_test.go

```diff
@@ -118,6 +118,22 @@ func TestTags(t *testing.T) {
 	suite.Run()
 }
 
+func TestFilterFeatureWithTags(t *testing.T) {
+	suite := NewSuite(t, WithFeaturesPath("features/filter_tags_*.feature"), WithTags([]string{"@run-this"}))
+	c := false
+
+	suite.AddStep(`the test should pass`, func(_ StepTest, _ Context) {
+		c = true
+	})
+	suite.AddStep(`fail the test`, fail)
```

Comment:
This step is part of a scenario that should not be executed since it does not have `@run-this` tag.